app/javascript/lib/cluster-utils.js

```diff
@@ -42,7 +42,8 @@ export function annotationKeyProperties(annotation) {
     name: annotation.name,
     type: annotation.type,
     scope: annotation.scope,
-    id: annotation.id
+    id: annotation.id,
+    isDisabled: annotation.scope == 'invalid'
```

Comment:
Safer to use '===' rather than '==' here

Reply:
fb73f5e